services/spiget/spiget-base.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const Joi = require('joi')
+const BaseJsonService = require('../base-json')
+
+const resourceSchema = Joi.object({
+  downloads: Joi.number().required(),
+  file: Joi.object({
+    size: Joi.number().required(),
+    sizeUnit: Joi.string().required(),
+  }).required(),
+  testedVersions: Joi.array(),
+  rating: Joi.object({
+    count: Joi.number().required(),
+    average: Joi.number().required(),
+  }).required(),
+}).required()
+
+const documentation = `
+<p>You can find your resource ID in the url for your resource page.</p>
+<p>Example: <code>https://www.spigotmc.org/resources/essentialsx.9089/</code> - Here the Resource ID is 9089.</p>`
+
```

Comment:
You mentioned before that `spiget` is the API platform for the service `spigot` or `spigotmc` right? If so what do you think about also adding `spigot` and/or `spigotmc` as keywords (can do it here in the base class the same way you added documentation) so that the badges will also show up on search results for `spigot` etc.?